readme.md

```diff
@@ -12,25 +12,43 @@
+As we write client-side Javascript -- javascript that is executed in *the browser* -- it is very important to keep the user's actions in mind when designing our app's UI.
 
 For example, let's say we have a single button on our landing page, we need to write some code that will execute whenever a user clicks on that button.
 
 ### What is Asynchronicity?
 
-Javascript typically will run top-to-bottom. We as developers, however, have no idea when the code related to the button click will actually be executed. It's totally dependent on the user. Therefore, we need to write code that will execute asynchronously -- or in other words, outside of the typical top-to-bottom document flow -- and not hold up the rest of our application.
+Javascript typically will run top-to-bottom. We as developers, however, have no idea when the code related to the button click will actually be executed.
+It's totally dependent on the user.
+Therefore, we need to write code that will execute asynchronously -- or in other words, outside of the typical top-to-bottom document flow -- and not hold up the rest of our application.
+
+> The term asynchronicity, like many in programming may be unduly intimidating. It simply means "not happening at the same time".
+> We have actually seen this at play in our own code regularly when working at the REPL and not found it to be intuitive possibly to the point of being mundane.
+> The reason for using terms like asynchronicity is not to sound smart or create a barrier to entry, but so that we can be very exact when we talk about programming.
 
-Javascript as a language was built with this problem in mind and has spawned many more solutions that have been introduced through libraries, packages and frameworks. Together, these tools provide some abstractions to interact with events. As such, we as developers can write code to listen for and respond to these events.
+Javascript as a language was built with this problem in mind and has spawned many more solutions that have been introduced through libraries, packages and frameworks as well as default language features.
+Together, these tools provide some abstractions to interact with events. As such, we as developers can write code to listen for and respond to these events.
 
 Today, we will get practice writing the underlying code responsible for adding behavior to a webpage with jQuery.
 
 ### jQuery and Vanilla Javascript
 
-Note that we are using jQuery for this lesson. jQuery is a **library** written in javascript that allows us to do more with less code. Check out [this](http://youmightnotneedjquery.com/) website for side-by-side comparisons of jQuery and vanilla javascript.
+Note that we are using jQuery for this lesson. jQuery is a **library** written in javascript that allows us to do more with less code and hides some frustrating or counterintuitive behavior of the DOM.
+
+> What code that hides unimportant details of something called?
```

Comment:
I think this sentence got garbled somehow. How's this?:

"What term do we use when code hides unimportant details from us?"